types/formidable/VolatileFile.d.ts

```diff
@@ -1,5 +1,14 @@
-import PersistentFile = require("./PersistentFile");
+import { EventEmitter } from 'events';
```

Comment:
```suggestion
import { EventEmitter } from 'node:events';
```
(keep with upstream)

Reply:
Fixed :)